src/extension/settingsHelper.ts

```
@@ -63,4 +67,22 @@ export class SettingsHelper {
                 return settingsJson["typescript.tsdk"] || "";
             });
     }
+
+    /**
+     * Reads and parses the launch.json file.
+     */
+    public static readLaunchJson(): Q.Promise<any> {
+        let fileSystem = new FileSystem();
+        let launchJsonPath = SettingsHelper.launchJsonPath;
+
+        return fileSystem.exists(launchJsonPath)
+            .then(exists => {
+                if (exists) {
+                    return fileSystem.readFile(launchJsonPath)
+                        .then(content => { return JSON.parse(content); });
+                } else {
+                    return null;
```

I'd reject the promise instead of returning null